spec/index.bs

```diff
@@ -2088,8 +2101,46 @@ enum GPUTextureAspect {
         </pre>
 
         **Returns:** |view|, of type {{GPUTextureView}}.
-
-        Issue: write definition. |this| |descriptor| |view|
+        1. [=resolve GPUTextureViewDescriptor defaults=]
+        1. If any of the following conditions are unsatisfied, generate a validation
+            error and return an [=invalid=] {{GPUTextureView}}.
+            <div class=validusage>
+                - 0 &le; {{GPUTextureViewDescriptor/baseMipLevel}} &lt; {{GPUTextureDescriptor/mipLevelCount}}
+                - 1 &le; {{GPUTextureViewDescriptor/mipLevelCount}} &le; {{GPUTextureDescriptor/mipLevelCount}} - {{GPUTextureViewDescriptor/baseMipLevel}}
```

Comment:
GPUTextureViewDescriptor/mipLevelCount and GPUTextureDescriptor/mipLevelCount are indistinguishable in the rendered text. Needs to be `|something|.{{GPUTextureViewDescriptor/mipLevelCount}}` and so on.